python/metatensor_torch/tests/atomistic/ase_calculator.py

```diff
@@ -177,6 +178,47 @@ def test_selected_atoms(tmpdir, model, atoms):
     assert np.allclose(expected, first_energies + second_energies)
 
 
+def test_compute_energy(tmpdir, model, atoms):
+    ref = atoms.copy()
+    ref.calc = ase.calculators.lj.LennardJones(
+        sigma=SIGMA, epsilon=EPSILON, rc=CUTOFF, ro=CUTOFF, smooth=False
+    )
+
+    path = os.path.join(tmpdir, "exported-model.pt")
+    model.save(path)
+    calculator = MetatensorCalculator(path, check_consistency=True)
+
+    energy = calculator.compute_energy(atoms)["energy"]
+    assert np.allclose(ref.get_potential_energy(), energy)
+
+    results = calculator.compute_energy(atoms, compute_forces_and_stresses=True)
+    assert np.allclose(ref.get_potential_energy(), results["energy"])
```

Comment:
shouldn't this be strictly equal?

Reply:
The calculator here is not the same model, it's the LJ implementation of ASE

Reply:
ah, right